Bot/TynibotHost.cs

```diff
@@ -87,6 +85,9 @@ public TynibotHost()
                 await Client.LoginAsync(TokenType.Bot, this.Settings.BotToken);
                 await Client.StartAsync();
 
+                // Bootstrap the CEA Data (Otherwise first response will timeout)
+                PlayCEAStats.RequestManagement.LeagueManager.Bootstrap();
```

Comment:
Moved so that it's not blocking to the Ready check - but will bootstrap once at startup prior to entering wait.